docs/references.rst

```diff
@@ -16,3 +16,6 @@ Reference
     Network requirements <references/network_requirements.md>
     Personal Package Archives (PPAs) <references/ppas.md>
     Support matrix <references/support_matrix.md>
+    What is a CVE <references/what_is_a_cve.md>
+    What is a USN <references/what_is_a_usn.md>
+    What are related USNs <references/what_are_related_usns.md>
```

Comment:
I think these are "explanation" rather than reference. I also think these can all be combined into one explanation doc titled something like "What's the difference between CVEs and USNs?"

Reply:
Agreed. All three are explanations and can be part of a single explanation page. If someone doesn't know what a USN is, it's fair to assume they also wouldn't know what a CVE is so not much need to separate the three. I'd call it something like "About CVEs/USNs" or "CVEs and USNs explained" or something like that.